src/assets/css/App.scss

```diff
@@ -124,7 +122,7 @@ $primary-color: black;
   padding: 2px;
   color: grey;
   background: rgb(233, 233, 233);
-  transition: all .3s;
+  transition: all 0.3s;
```

Comment:
You don't have to add the Zero on the transition style attribute

Reply:
Yeah it works but its not the convention....